src/FixedDataTable.js

```diff
@@ -184,6 +184,24 @@ var FixedDataTable = createReactClass({
     rowHeightGetter: PropTypes.func,
 
     /**
+     * Pixel height of cells unless `cellHeightGetter` is specified and returns
+     * different value.
+     */
+    cellHeight: PropTypes.number,
```

Comment:
I am sorry that if I use expandedContentHeight, there is too much logic and I can't solve it, I think rowHeight = cellHeight + expandedContentHeight will be more easily. if no cellHeight, then cellHeight = rowHeight. I very want expanded feature if you can help add expandedContentHeight, I'll always appreciate that